Add my information to lab website

lcsanford opened the pull request:
Adding info about me to the data file

lcsanford committed:
Update home.md

lcsanford committed:
Create test.txt

lcsanford committed:
Add files via upload

lcsanford committed:
Add files via upload

lcsanford committed:
Add files via upload

lcsanford committed:
Add files via upload

lcsanford committed:
Update home.md

lcsanford committed:
Rename SPIRES_slider/Forest_theory.png to images/Spires_slider/Forest_theory.png

lcsanford committed:
Rename images/Spires_slider/Forest_theory.png to images/SPIRES_slider/Forest_theory.png

lcsanford committed:
Add files via upload

lcsanford committed:
Delete SPIRES_slider directory

lcsanford committed:
Update home.md

lcsanford committed:
Update home.md

lcsanford committed:
Add files via upload

lcsanford committed:
Update home.md

lcsanford committed:
Update home.md

lcsanford committed:
Add files via upload

lcsanford committed:
Update research.md

lcsanford committed:
Update publications.md

lcsanford committed:
Update publist.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update team_members.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update team_members.yml

lcsanford committed:
Update team_members.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update students.yml

lcsanford committed:
Update students.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update students.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update team_members.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update team_members.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update team_members.yml

HungryAmoeba committed:
Update students.yml

Sanford-Lab committed:
Merge pull request #9 from HungryAmoeba/patch-1

Update students.yml

Sanford-Lab committed:
Update homelay.html

Sanford-Lab committed:
Add files via upload

Sanford-Lab committed:
Delete Group05022015.jpg

Sanford-Lab committed:
Delete Group05022015_red.jpg

Sanford-Lab committed:
Delete Group2015.png

Sanford-Lab committed:
Delete Group2015_red.jpg

Sanford-Lab committed:
Delete Logos_h.jpg

Sanford-Lab committed:
Delete News_M2PG.png

Sanford-Lab committed:
Delete News_Maarten.png

Sanford-Lab committed:
Delete dummy.png

Sanford-Lab committed:
Delete logo2.png

Sanford-Lab committed:
Delete mm-browser-mockups.png

Sanford-Lab committed:
Delete mm-layout-examples.png

Sanford-Lab committed:
Delete Logo_ERC.jpg

Sanford-Lab committed:
Delete Logo_Leiden.jpg

Sanford-Lab committed:
Delete Logo_NWO.jpg

Sanford-Lab committed:
Delete Logo_Nanofront.jpg

Sanford-Lab committed:
Update _config.yml

Sanford-Lab committed:
Add files via upload

Sanford-Lab committed:
Update home.md

Sanford-Lab committed:
Update allnews.md

Sanford-Lab committed:
Update mini_lecture_series.md

Sanford-Lab committed:
Update openings.md

Sanford-Lab committed:
Update pictures.md

Sanford-Lab committed:
Update team.md

Sanford-Lab committed:
Update header.html

Sanford-Lab committed:
Update footer.html

Sanford-Lab committed:
Update home.md

Sanford-Lab committed:
Delete favicon.ico

Sanford-Lab committed:
Add files via upload

Sanford-Lab committed:
Update home.md

Sanford-Lab committed:
Delete favicon.ico

Sanford-Lab committed:
Add files via upload

Sanford-Lab committed:
Update home.md

Sanford-Lab committed:
Delete favicon.ico

Sanford-Lab committed:
Update home.md

Sanford-Lab committed:
Add files via upload

Sanford-Lab committed:
Update team_members.yml

Sanford-Lab committed:
Update publications.md

lcsanford committed:
Update publist.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update publist.yml

lcsanford committed:
Update publist.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update news.yml

lcsanford committed:
Update news.yml

lcsanford committed:
Update publist.yml

lcsanford committed:
Update publist.yml

lcsanford committed:
Update openings.md

lcsanford committed:
Update alumni_members.yml

lcsanford committed:
Add files via upload

lcsanford committed:
Update alumni_members.yml

lcsanford committed:
Update team.md

lcsanford committed:
Update home.md

lcsanford committed:
Update team.md

lcsanford committed:
Update team_members.yml

lcsanford committed:
Update students.yml

lcsanford committed:
Update team.md

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Update home.md

lcsanford committed:
Add files via upload

lcsanford committed:
Delete SPIRES_logo.png

lcsanford committed:
Add files via upload

lcsanford committed:
Update research.md

lcsanford committed:
Delete images/respic directory

lcsanford committed:
Create test.txt

lcsanford committed:
Add files via upload

lcsanford committed:
Update home.md

lcsanford committed:
Update home.md

lcsanford committed:
Update homelay.html

lcsanford committed:
Update home.md

lcsanford committed:
Update news.yml

lcsanford committed:
Update news.yml

lcsanford committed:
Update homelay.html

lcsanford committed:
Update homelay.html

lcsanford committed:
Update homelay.html

lcsanford committed:
Update homelay.html

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Update homelay.html

lcsanford committed:
Update homelay.html

lcsanford committed:
Update homelay.html

lcsanford committed:
Update home.md

lcsanford committed:
Update home.md

lcsanford committed:
Update home.md

lcsanford committed:
Update home.md

lcsanford committed:
Update research.md

lcsanford committed:
Add files via upload

lcsanford committed:
Add files via upload

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Add files via upload

lcsanford committed:
Add files via upload

lcsanford committed:
Update research.md

lcsanford committed:
Delete network_detail.png

lcsanford committed:
Add files via upload

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Add files via upload

lcsanford committed:
Update research.md

lcsanford committed:
Update research.md

lcsanford committed:
Delete Brazil.tiff

lcsanford committed:
Add files via upload

lcsanford committed:
Update research.md

lcsanford committed:
Update openings.md

lcsanford committed:
Update openings.md

lcsanford committed:
Add files via upload

lcsanford committed:
Update openings.md

lcsanford committed:
Update openings.md

megan-k-ayers committed:
Update research.md

Updated the Remote Control and Discovering Persuasive Concepts project descriptions.

lcsanford committed:
Merge pull request #10 from Sanford-Lab/ma-research-descript-updates

Update research.md

Sanford-Lab committed:
new Luke headshot

Sanford-Lab committed:
Update team_members.yml

Sanford-Lab committed:
Update team_members.yml

Sanford-Lab committed:
Update alumni_members.yml

Sanford-Lab committed:
Update team_members.yml

Sanford-Lab committed:
Update alumni_members.yml

laurenoliver95 commented:
name: Tala
![tala.jpeg](https://github.com/mpa139/allanlab/assets/145479711/11bc3e14-e0c1-477f-b564-dcd82cbdf949)

  photo: tala.jpeg
  info: Lab dog
  number_educ: 1
  education1: Adult Level 1 at Petco

lcsanford committed:
add lab guide draft

AlbertogGarcia committed:
Add files via upload

AlbertogGarcia committed:
Update team_members.yml to include Albert

laurenoliver95 committed:
Update team_members.yml

laurenoliver95 committed:
Add files via upload

laurenoliver95 committed:
Update team_members.yml

Commit:
Add Owen's information to file

Commit:
Add Owen's photo

Commit:
fixed Owen's info

Sanford-Lab committed:
Merge pull request #11 from AlbertogGarcia/gh-pages

Adding info for Albert

Sanford-Lab committed:
Merge pull request #12 from laurenoliver95/gh-pages

update lab page

Sanford-Lab committed:
Merge pull request #13 from ocortner/add-owen-info

Add owen info

Sanford-Lab committed:
fix Tala

Sanford-Lab committed:
Update team_members.yml

Sanford-Lab committed:
Update team_members.yml

laurenoliver95 committed:
Add files via upload

laurenoliver95 committed:
Update team_members.yml

Sanford-Lab committed:
Merge pull request #15 from laurenoliver95/laurenoliver95-patch-2

Add files via upload

Sanford-Lab committed:
Merge branch 'gh-pages' into laurenoliver95-patch-3

Sanford-Lab committed:
Merge pull request #14 from laurenoliver95/laurenoliver95-patch-3

Update team_members.yml

Sanford-Lab committed:
Update team_members.yml

graciahw committed:
Update team_members.yml

graciahw committed:
Add files via upload

graciahw committed:
Update team_members.yml

Sanford-Lab committed:
Merge pull request #16 from graciahw/gh-pages

Adding Grace

Sanford-Lab committed:
Update team_members.yml

graciahw committed:
Add files via upload

Sanford-Lab committed:
Merge pull request #17 from graciahw/gh-pages

Adding Grace's pic

Sanford-Lab committed:
Update team_members.yml

remove albert

Sanford-Lab committed:
Update alumni_members.yml

update alums

Sanford-Lab committed:
Update news.yml

Sanford-Lab committed:
Update openings.md

maikepf committed:
Add files via upload

maikepf committed:
Update alumni_members.yml

moved Allie to alumni

maikepf committed:
Update team_members.yml

Added Sarah, Mariam, Apurva and Maike profiles, moved Allie to alumni, updated titles for Eliana and Megan, added personal website for Owen, updated education for eliana

maikepf committed:
Update team_members.yml

fix file extension for profile images: sarah, mariam, maike

maikepf committed:
Add files via upload

maikepf committed:
Update team.md

removed 'phd students' from 'looking for ... to join the team' at beginning of page

maikepf committed:
Update team_members.yml

changes 'B.S.' titles to all be 'B.Sc.'

maikepf committed:
Add files via upload

maikepf committed:
Add files via upload

Added rationalizing figure for aclanthology pub

maikepf committed:
Update publist.yml

added "Discovering influential text using convolutional neural networks"

maikepf committed:
Update team_members.yml

fixed apurva's email address

maikepf committed:
Add files via upload

maikepf committed:
Update team_members.yml

added new profile picture for Lauren

maikepf committed:
Update team_members.yml

maikepf committed:
Add files via upload

maikepf committed:
Update team_members.yml

added mike zhang

maikepf committed:
Update team_members.yml

maikepf committed:
Update team_members.yml

maikepf committed:
Update alumni_members.yml

maikepf committed:
Update team_members.yml

Added Reid Lewis to team members

maikepf committed:
Add Reid Lewis picture

maikepf committed:
Update team_members.yml

maikepf committed:
Add files via upload

maikepf committed:
Update team_members.yml

maikepf committed:
Update team_members.yml

changed order of reid's degrees

maikepf committed:
Update team_members.yml

fixed inconsistency in master degree names

maikepf committed:
Update team_members.yml

fixed inconsistency in degree name

maikepf committed:
Update team_members.yml

maikepf committed:
Update team_members.yml

Move Sarah C to alumni

maikepf committed:
Update alumni_members.yml

Move Sarah C to alumni

maikepf committed:
Update home.md

Removed "PhD students" from vacancy announcements on landing page

maikepf committed:
Add files via upload

Add Lara headshot

maikepf committed:
Update team_members.yml

Add Lara Briggs to team page

maikepf committed:
Add files via upload

add jonathan's headshot

maikepf committed:
Update team_members.yml

add jonathan elkobi toteam profiles

maikepf committed:
Update team_members.yml

fixed typo

maikepf committed:
Update team_members.yml

added graduation year for lara briggs

Copilot committed:
Initial plan

Sanford-Lab committed:
Merge pull request #18 from Sanford-Lab/copilot/add-recent-publications-luke-sanford

Initial exploration for adding recent Luke Sanford publications

lcsanford committed:
Update lab website with recent publications and working papers

- Add 7 recent working papers to research page with full abstracts:
  - Adversarial Debiasing for Unbiased Parameter Recovery (arXiv)
  - Elevating Spatial Evaluation: Satellite-Driven Confounder Adjustment (SSRN)
  - Causal Carbon: Baselines and Additionality with Potential Outcomes (OSF)
  - Strategic behavior in jurisdictional REDD+ (SSRN)
  - Large but Restricted Tree Cover Loss Associated with Mexico's Mayan Train (Working Paper)
  - The Political Economy of Conflict and Deforestation: Evidence from the Philippines (Working Paper)
  - Electoral Deforestation Cycles: A Subnational Analysis (Working Paper)

- Add published paper: Estimating Missingness and Delay in Court Data (Sociological Methods & Research)
- Update copyright year from 2022 to 2025
- Remove placeholder publications and clean up publist.yml

lcsanford committed:
Resolve merge conflict in students.yml - keep active students from remote

lcsanford committed:
Add new working paper and update paper statuses

- Add 'The Political Contours of Land Reform: Conditional Impacts of Tenure Formalization on Socio-Economic and Environmental Outcomes' (Under Review, July 2025)
- Update 'Adversarial Debiasing for Unbiased Parameter Recovery' status to Under Review, February 2025
- Update 'Elevating Spatial Evaluation: Satellite-Driven Confounder Adjustment' status to Revise and Resubmit, 2025

lcsanford committed:
update pubs

lcsanford committed:
update working papers

lcsanford committed:
add vincent paper

lcsanford committed:
updated pubs

lcsanford committed:
fix YAML parsing error

lcsanford committed:
change order of pubs

lcsanford committed:
Add new lab picture for November 2025 and clean up pictures page layout

lcsanford committed:
Update lab website with new publications and revise existing entries

- Add new working paper: 'The Impact of Climate Change on Agricultural Productivity' (In Progress)
- Update status of 'Estimating Missingness and Delay in Court Data' to Published, 2025
- Revise publication details for 'Causal Carbon: Baselines and Additionality with Potential Outcomes'

lcsanford committed:
home page

lcsanford committed:
add ls cv

Sanford-Lab committed:
Add files via upload

Sanford-Lab committed:
Comment out website field in team_members.yml

Commented out the website field for Luke C Sanford.

Sanford-Lab committed:
Update alumni member info for Sarah Castle

Sanford-Lab committed:
Remove Megan Ayers from team_members.yml

Removed Megan Ayers from the team members list.

Sanford-Lab committed:
Add Megan Ayers to alumni members list

maikepf committed:
Add files via upload

add Kexin headshot

maikepf committed:
Update team_members.yml

Add Kexin to team members page
